### PR TITLE
#3311 - Student financial information content

### PR DESCRIPTION
**Pull request by katrina-aot-git**
Update the content of the financial section of applications.
image
image

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/sfaa2022-23.json [hunk 1]
```diff
@@ -31259,7 +31259,7 @@
                   "tableView": false,
                   "inputType": "text",
                   "inputMask": "",
-                  "label": "Please estimate your weekly transportation costs.",
+                  "label": "Please estimate your weekly transportation costs:",
```
Comment (andrepestana-aot) [inline sources/packages/forms/src/form-definitions/sfaa2022-23.json:31262]: In the PDF is says this question should be removed.

Comment (andrepestana-aot) [reply sources/packages/forms/src/form-definitions/sfaa2022-23.json:31262]: There's no AC for that in the ticket. Please just revert the changes in that line.

**Review by andrepestana-aot — COMMENTED** [review]
Looks good. Just left a minor comment.

File: sources/packages/forms/src/form-definitions/sfaa2022-23.json (hunk 1 above)
Comment (katrina-aot-git) [reply sources/packages/forms/src/form-definitions/sfaa2022-23.json]: Updated

**Review by andrepestana-aot — APPROVED** [review]
Good job!

**Pull request merged**